### PR TITLE
Complete To do list app

### PR DESCRIPTION
**Pull request by jubaer919**
- Project is setup with web-pack
- Html is rendering from plugins
- project design is simple and followed the instruction 
- every li has checkbox, index and paragraph

### CODE REVIEW

File: src/toDo.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  // to do list function
+  function toDoList() {
+    const listContainer = document.querySelector('.to-do-list');
+
+    tasks.forEach((task) => {
+      // Create list item element
+      const listItem = document.createElement('li');
+      listItem.classList.add('list-items');
+      listItem.innerHTML = `<input type='checkbox' id='list-checkbox' name='checkbox'><p class='list-paragraph'>${task.description}</p><img src='${threedot}'  alt='#' class='list-img'>`;
```
Comment (AdnanOlarmmi) [inline src/toDo.js]: - [ ] Nice one dynamically rendering the tasks, but kindly note that the `checkbox` or in your case the  `checked` property of the checkbox  should be rendered dynamically also with respect to the `completed` in each objects, meaning it should be checked on `completed: true` and unchecked on completed false...

You could do something like. `<input class='checkbox' type="checkbox" ${task.completed && 'checked'}/>` or more clearer with ternary operator `<input class='checkbox' type="checkbox" ${elem.completed? 'checked' : ''}/>`

the `checked` property is like the `required` property, but what it does is that it triggers the checked icon of the checkbox, when it is present.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @jubaer919, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Task rendering dynamically using JavaScript.
- Web pack implemented properly.
- App looks like wireframe.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @jubaer919, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Task rendering dynamically using JavaScript.
- Web pack implemented properly.
- App looks like wireframe.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @jubaer919 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All project requirements implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**